ci-operator/config/openshift/azure-service-operator/OWNERS

```diff
@@ -0,0 +1,19 @@
+# WARNING: This file is autogenerated. DO NOT edit by hand!!!
+# To update this file please contact an existing owner and request
+# to be added to the owners-gen input data.
+approvers:
+- JoelSpeed
+- elmiko
+- nrb
+- RadekManak
```

Comment:
Can you please add me here too. Since this is CAPI related I'd like to be in the approvers list

Reply:
It should be, but afaik there's nothing that syncs them for us. Is that incorrect?

Reply:
I added myself to the owners in the spreadsheet. But yeah, maybe we could run the ownership sync in CI on a periodic?